lib/stages/VisSharedMemWriter.cpp

```diff
@@ -211,6 +215,7 @@ bool VisSharedMemWriter::add_sample(const visFrameView& frame, time_ctype t, uin
         INFO("Dropping integration as buffer (FPGA count: {:d}) arrived too late (only accepting "
              "new times greater than {:d})\n",
              t.fpga_count, max_time.fpga_count);
+        dropped_frame_counter.labels({std::to_string(t.fpga_count)}).inc();
```

Comment:
Not sure exactly what to call this reason. `reason=order`? As I guess the issue is fundamentally that some data came out of order here, and we got some data from a timesample ahead of this one, before getting anything from this time.